.github/workflows/Tests.yml

```
@@ -237,33 +238,28 @@ jobs:
-        fi
-    - name: deepsource for ${{ matrix.python-version }} on ${{ matrix.os }}
-      if: ${{ !cancelled() && runner.os != 'Windows' }}
-      id: coverage-deepsource-upload
-      shell: bash
-      env:
-        DEEPSOURCE_DSN: ${{ secrets.DEEPSOURCE_DSN }}
-      run: |
-        ./bin/deepsource report --analyzer test-coverage --key python --value-file ./coverage.xml 2>/dev/null || true ;
+    - id: upload-test-tools
+      if: ${{ !cancelled() }}
+      uses: ./.github/actions/test-reporter-upload
```

actionlint (1.7.7)

could not parse action metadata in "/home/jailuser/git/.github/actions/test-reporter-upload": yaml: line 274: could not find expected ':'

(action)